pkg/broker/handler/pool_test.go

```diff
@@ -74,17 +74,16 @@ func TestSyncPool(t *testing.T) {
 
 		ch <- struct{}{}
 		syncPool.verifySyncOnceCalled(t)
-		assertProbeCheckResult(t, p, true)
-
-		// Intentionally causing a failed check.
-		time.Sleep(time.Second)
-		assertProbeCheckResult(t, p, false)
+		// False because authentication check will fail.
+		assertProbeCheckResult(t, p, false, "healthz")
```

Comment:
Should we also have positive test cases?

Reply:
done